src/Microsoft.AspNetCore.Razor.Language/Legacy/BackgroundParser.cs

```diff
@@ -67,37 +69,43 @@ protected virtual void OnResultsReady(DocumentParseCompleteEventArgs args)
             }
         }
 
-        internal static bool TreesAreDifferent(RazorSyntaxTree leftTree, RazorSyntaxTree rightTree, IEnumerable<TextChange> changes)
-        {
-            return TreesAreDifferent(leftTree, rightTree, changes, CancellationToken.None);
-        }
-
+#pragma warning disable CS0612 // Type or member is obsolete
```

Comment:
I don't want to make incremental updates to `RazorEditorParser`, I'm trying to replace it completely.